Client/src/i18n/ru/admin.js

```diff
@@ -10,7 +10,8 @@ export default {
   AdminMenu: {
     categoriesAdmin: "Категории",
       rolesPermissions: "Настройка прав",
-      rolesUsers: "Группы пользователей"
+      rolesUsers: "Группы пользователей",
```

Comment:
Should we also add en translation for new params?

Reply:
> Should we also add en translation for new params?
Localization system has been updated, my next PR (along with the site translation) will have a translation this label